### PR TITLE
new icon: norg (original, plain)

### PR DESCRIPTION
**Pull request by BinxDot**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

## **Additional Information**
Neorg (more specifically its underlying format, `norg`) is a relatively new file markup format aimed at being a successor to markdown and org.

## Link to prove your SVG is correct and up-to-date.

norg-original: https://raw.githubusercontent.com/nvim-neorg/neorg/main/res/neorg.svg
norg-plain: https://raw.githubusercontent.com/BinxDot/devicon/feat-icon/neorg/icons/norg/norg-plain.svg

### CODE REVIEW

**Comment by Snailedlt** [comment]
Is this the same thing?
https://github.com/nvim-neorg/neorg

If so, maybe this icon could be added too as the original version?
https://github.com/nvim-neorg/neorg/blob/main/res/neorg.svg

**Comment by BinxDot** [comment]
Hi, thanks for replying!

> Is this the same thing?
https://github.com/nvim-neorg/neorg

Yes, it indeed is (should have mentioned that, sorry).

> If so, maybe this icon could be added too as the original version?
https://github.com/nvim-neorg/neorg/blob/main/res/neorg.svg

Do you want me to add that icon as the `original` svg, and move the existing one to the `plain` version? It's my first time contributing here so I'm a bit confused 😅.

**Comment by Snailedlt** [comment]
@BinxDot
Okey :)
Yes please, do that!
Also if there are any wordmark or line versions you can add those as well, but no problem if there aren't any

**Comment by BinxDot** [comment]
@Snailedlt I have fixed it, please do let me know if I've messed up anything, not too experienced with git haha

File: devicon.json
Comment (Snailedlt) [inline devicon.json:4757]: Please add the altnames property as explained [here](https://github.com/devicons/devicon/wiki/Updating-the-%60devicon.json%60) and shown [here](https://github.com/devicons/devicon/wiki/Example-of-Submitting-Icons-to-the-Repository)

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Good start for a first time contribution. devicon.json is only missing alnames as far as I can tell. 

As for the icons, they look good, but the code is a bit messy. You should take a look at our [SVG Standards](https://github.com/devicons/devicon/wiki/SVG-Standards), and try to use some of our [recommended tools](https://github.com/devicons/devicon/wiki/Recommended-resources-and-tools) for resizing and optimizing the SVG's.

It might be good for you to go through the [overview on submitting icons](https://github.com/devicons/devicon/wiki/Overview-on-Submitting-Icons) wiki page for your first contribution too, so you get the full picture :)

Try and do some of the changes, and if you have any trouble, you can always ask me or some other supporter/maintainer to take a look again by requesting a new review :)

File: devicon.json
Comment (BinxDot) [reply devicon.json:4757]: Added it, thanks for reminding :)

**Comment by BinxDot** [comment]
Sorry for the delay, was busy!

> As for the icons, they look good, but the code is a bit messy. You should take a look at our [SVG Standards](https://github.com/devicons/devicon/wiki/SVG-Standards), and try to use some of our [recommended tools](https://github.com/devicons/devicon/wiki/Recommended-resources-and-tools) for resizing and optimizing the SVG's.

As for this, I have used [svgviewer.dev](https://www.svgviewer.dev/) to resize and optimize both the SVG's. Can you please review everything once again when you have the chance? Thanks!

File: icons/norg/norg-original.svg
Comment (Panquesito7) [inline icons/norg/norg-original.svg:31]: I optimized the icon using [SVG Viewer](https://svgviewer.dev). 🙂
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><defs><linearGradient id="a" gradientUnits="userSpaceOnUse" x1="205.272" y1="136.063" x2="800.726" y2="855.023" gradientTransform="matrix(.125 0 0 -.125 0 128)"><stop offset="0" style="stop-color:#33aae6;stop-opacity:1"/><stop offset=".992" style="stop-color:#4b3892;stop-opacity:1"/></linearGradient><linearGradient id="b" gradientUnits="userSpaceOnUse" x1="531.93" y1="357.426" x2="693.4" y2="357.426" gradientTransform="matrix(.125 0 0 -.125 0 128)"><stop offset="0" style="stop-color:#3d85c9;stop-opacity:1"/><stop offset="1" style="stop-color:#4a4198;stop-opacity:1"/></linearGradient><linearGradient id="c" gradientUnits="userSpaceOnUse" x1="531.92" y1="288.905" x2="944.488" y2="288.905" gradientTransform="matrix(.125 0 0 -.125 0 128)"><stop offset="0" style="stop-color:#61c67c;stop-opacity:1"/><stop offset="1" style="stop-color:#366794;stop-opacity:1"/></linearGradient><linearGradient id="d" gradientUnits="userSpaceOnUse" x1="843.48" y1="404.53" x2="969.756" y2="404.53" gradientTransform="matrix(.125 0 0 -.125 0 128)"><stop offset="0" style="stop-color:#2f6b7f;stop-opacity:1"/><stop offset="1" style="stop-color:#325d87;stop-opacity:1"/></linearGradient><linearGradient id="e" gradientUnits="userSpaceOnUse" x1="513.323" y1="718.004" x2="900.784" y2="143.708" gradientTransform="matrix(.125 0 0 -.125 0 128)"><stop offset="0" style="stop-color:#3b7fc4;stop-opacity:1"/><stop offset=".467" style="stop-color:#90b1df;stop-opacity:1"/><stop offset=".764" style="stop-color:#7381c0;stop-opacity:1"/><stop offset="1" style="stop-color:#5f5fab;stop-opacity:1"/></linearGradient></defs><path style="stroke:none;fill-rule:nonzero;fill:url(#a)" d="M111.113 107.7c46.95-64.388-36.812-96.177-72.226-40.263-4.473-7.449-6.375-17.101-2.438-31.8C20.074 54.687 18 74.488 35.863 95.336c14.25-26.688 32.188-31.125 50.825-31.211l-.012 13.625c-8.438-.852-14.227 5.477-20.188 11.227-9.875 7.347-18.375 18.023-23.937 35.261C-26.961 98.113-4.414-4.176 69.3.512c-7.602 8.113-14.114 16.625-13.727 27.687 56.664-40.097 96.301 37.574 55.95 79.102"/><path style="stroke:none;fill-rule:nonzero;fill:url(#b)" d="M86.648 88.96c-1.375-11.147-12.312-8.136-20.16.017 5.961-5.75 11.739-12.079 20.188-11.227l-.028 11.21"/><path style="stroke:none;fill-rule:nonzero;fill:url(#c)" d="m86.664 88.977-.016 14.835c15.403-7.511 22.454-31.789 21.84-47.976 13.114 17.465 11.399 34.578 3.024 51.465h.011c-11.449 12.16-27.734 20-45.023 20.636l-.012-38.96c7.848-8.153 18.785-11.165 20.176 0"/><path style="stroke:none;fill-rule:nonzero;fill:url(#d)" d="M111.512 107.3c8.375-16.874 10.074-34-3.024-51.464-.3-3.125-1.152-6.125-3.05-8.273 19.425 13.687 20.273 40.124 6.074 59.738"/><path style="stroke:none;fill-rule:nonzero;fill:url(#e)" d="M111.523 107.3c14.188-19.612 13.34-46.062-6.085-59.737-22.075-13.864-52.825-.012-66.551 19.863 35.527-55.989 119.125-24.028 72.226 40.261"/></svg>
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Comment (BinxDot) [reply icons/norg/norg-original.svg:31]: Oh, I did the "prettify" option to improve readability since the code was messy. My bad!

File: icons/norg/norg-plain.svg
Comment (Snailedlt) [inline icons/norg/norg-plain.svg:1]: Height is just `px` here. Not sure if it has any effect, but since we don't know what it does I'd suggest removing it so we don't get unexpected bugs down the line :)

Come to think of it, I think we can remove the width and height property for both icons. Not sure though, but worth trying maybe?

File: devicon.json
Comment (Snailedlt) [inline devicon.json:4774]: I can't find this color in any of the icons, did you perhaps forget to change it?
```suggestion
        "color": "#366794",
```

Couldn't find any official color for their branding, but imo one of the most prevalent color from the logo will do. The one suggested above is just a suggestion. You can pick one of the other colors in the logo if you want, or even better, an official branding color if you can find it

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Some minor changes I would make, other than that it looks good 😁
Just one more push and we're there 👌

File: icons/norg/norg-plain.svg
Comment (BinxDot) [reply icons/norg/norg-plain.svg:1]: Yeah you're right, I just removed the `height="px"` from the plain version

Though removing both width and height properties from both versions seemed to have put both icons in a different size.

File: devicon.json
Comment (BinxDot) [reply devicon.json:4800]: We actually got this color by getting an average of the colors in the official logo, but the one you have provided looks much better! Thanks again :)

Comment (Snailedlt) [reply devicon.json]: Ahh, that makes sense. You decide which one you choose then :)

**Comment by Snailedlt** [comment]
@BinxDot is it just me or is the icon on iconmoon being cut off ever so slightly on the right?

**Comment by Snailedlt** [comment]
> @BinxDot is it just me or is the icon on iconmoon being cut off ever so slightly on the right?

Doesn't look like this in the icon you uploaded, so something is happening with icomoon. Try adding in the height parameter again, and see if that helps. If not, I don't think it's a big deal anyways, since it's barely noticable

File: icons/norg/norg-plain.svg
Comment (Snailedlt) [inline icons/norg/norg-plain.svg:1]: See if this fixes the issue cropping issue on icomoon ```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128">
```

**Comment by BinxDot** [comment]
Looks like adding the height parameter worked!

File: devicon.json
Comment (BinxDot) [reply devicon.json]: So after a bit of deciding we've decided to go with `#4878be` which is a bit of a lighter color from the one you suggested

**Comment by Snailedlt** [comment]
> Looks like adding the height parameter worked!

Yeah! 😁
Still cutting off like 2 or 3 pixels, but it's not noricable unless you look for it, so it's all good :)

**Comment by BinxDot** [comment]
I've just made the primary color a bit lighter in the devicon.json :)

**Comment by Snailedlt** [comment]
Running bot peek one last time then :)

**Comment by Snailedlt** [comment]
> > Looks like adding the height parameter worked!
> 
> Yeah! grin Still cutting off like a 2 or 3 pixels wide chunk, but it's not noticable unless you look for it, so it's all good :)

This is probably just because of pixelation in the screenshot, the SVG preview on gothub sometimes get cut off in a similar manner

**Review by Snailedlt — APPROVED** [review]
Looks great! :heavy_check_mark: 

Thank you for your contribution! :rocket:

**Review by Panquesito7 — APPROVED** [review]
Amazing. Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Maintainable

**Pull request merged**
